packages/client/modules/meeting/components/SelectTemplate.tsx

```diff
@@ -41,6 +41,10 @@ const Button = styled(FloatingActionButton)({
   pointerEvents: 'all'
 })
 
+const UpgradeButton = styled(Button)({
+  padding: '10px 24px'
```

Comment:
Increased size of upgrade button following feedback [here](https://parabol.slack.com/archives/C042E8PGPD4/p1668552680439969?thread_ts=1668522709.276439&cid=C042E8PGPD4)